ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java

```diff
@@ -149,6 +149,7 @@ public CacheConfigHolder cacheConfigMap() {
 		cm.addDistMap(DIST_MAP_NAME_SAMPLING, 15);
 		cm.addDistMap(DIST_MAP_NAME_MONITORING, 15);
 		cm.addDistMap(DIST_MAP_NAME_AGENT, 10);
+		cm.addDistMap(CACHE_RECENTLY_USED_AGENTS, 1 * DAY);
```

Comment:
Please rename CACHE_RECENTLY_USED_AGENTS to DIST_ MAP_RECENTLY_USED_AGENTS

Reply:
Fixed